src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+
+    <a href="" class="t-ctaBtn">Buy ticket for {price}</a>
+    <span class="t-ctaNote">Price is Early Bird</span>
+
+  </article> -->
+  <WorkshopForm formEndpoint={endpointA11Y}>
+    <p class="t-ctaPitch">Get 9 hours, over 3 days, to refine your A11Y knowledge.</p>
+
+    <dl class="t-ctaPoints">
+      <dt class="time-todo">
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#calendar" />
+        </svg>
+        <span class="sr-only">When:</span>
+      </dt>
+      <dd>Soon</dd>
+
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+  </WorkshopForm>
+
+
+
+  <h2 class="t-title asH2">
+    F.A.Q.
+  </h2>
+
+  <div class="faqArea">
+    <ul class="faqList">
+    <li>
+        <Accordion summary="What does A11Y mean?">
+          A11Y stands for 'accessibility'.
+          The irony about the word "accessibility" itself is that it's not very easy to pronounce.
+          So, it was created this abbreviation: A + 11 characters + Y, which has a more friendly homophone of "ally".
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Will the workshop be recorded?">
+          No, it won't be recorded, but you’ll have access to all the materials shared.
+        </Accordion>
+      </li>
+
+
+      <li>
+        <Accordion summary="What materials are provided?">
+         <ul class="t-list">
+          <li>
+            The materials slides (+ 50 slides);
+          </li>
+          <li>
+            Codebase with all the mentioned resources;
+          </li>
+          <li>
+            Exercises, including the solutions explained;
+          </li>
+         </ul>       <!-- TODO list -->
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="I know A11Y. Will I learn something new?">
+          Oh, yeah. Even me, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gap.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is the workshop accessible?">
+          The video meeting can be captioned and the exercises briefings are fully accessible. If you have any accessibility concern, please reach out to me, and we can arrange it together.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="What screen reader will I use?">
+                        <!-- TODO list -->
+
+          You can use the one that suits you the best. I own a Mac, which means I’ll be using Voice Over.
```

Comment:
You can use the one that suits you best. I own a Mac, which means I’ll be using Voice Over.